supplementary_style_guide/glossary_terms_conventions/general_conventions/a.adoc

```diff
@@ -407,6 +418,17 @@ An Ansible task is a set of instructions to achieve a state defined, in its broa
 
 *See also*:
 
+[discrete]
+[[appstream-repository]]
+==== image:images/yes.png[yes] AppStream repository (noun)
+*Description*: The _AppStream repository_  distributes RHEL content as Application Streams and other components. These Application Streams include additional user-space applications, runtime languages, and databases for various workloads and use cases.
```

Comment:
Maybe, for parallelism and conciseness:
```suggestion
*Description*: The _AppStream repository_  distributes RHEL content as Application Streams and other components. Do not confuse the "AppStream repository" with the "Application Streams" that it distributes.
```
- Removed the extended description of "Application Streams," which is covered in its own entry.

Reply:
Thank you :-)